### PR TITLE
Fixes hardcoded 'water' messages when drowning in liquid

### LINKED ISSUE #21318: Drowning or swallowing in liquid pools says you're doing it in water, regardless of the actual reagent.
## Testmerges:

I don't believe this issue was caused by any testmerges.

## Reproduction:

1. Fill up a pool with any liquid at all
2. Slip or drown in it. It will always say you're swallowing/choking on water, regardless of what reagent you're actually in.

I did take a look at liquid_effect.dm and life.dm, and i discovered that it was hardcoded to say it was water. I did try to fix it, but i couldn't figure out how to get it working on life.dm.

### PR DESCRIPTION
**Pull request by vinylspiders**
## About The Pull Request

Fixes https://github.com/Skyrat-SS13/Skyrat-tg/issues/21318

Added a proc for `liquid_turf` called `reagents_to_text()` which will return a string of the liquids listed out in plain english and replaced the hardcoded 'water' messages with this proc.

## How This Contributes To The Skyrat Roleplay Experience

Fixes a slightly immersion ruining bug.

## Proof of Testing

<details>
<summary>lists them in grammatically correct english</summary>

image

image
 
image
 
</details>

## Changelog

:cl:
fix: drowning in liquid that is not water will now give the appropriate message based on what kind of liquid it is
/:cl:

### CODE REVIEW

**Comment by tf-4** [comment]
can you decapitalise?

**Review by Gandalf2k15 — APPROVED** [review]
another good improvement, epic!

**Comment by vinylspiders** [comment]
> can you decapitalise?

Sure, don't see why not.

**Review by tf-4 — APPROVED** [review]
epic
